app/src/main/res/layout/frag_filesystem_edit.xml

```diff
@@ -22,13 +22,63 @@
 
     </android.support.design.widget.TextInputLayout>
 
+    <android.support.design.widget.TextInputLayout
+        android:id="@+id/text_input_layout_filesystem_username"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:padding="8dp"
+        app:layout_constraintTop_toBottomOf="@+id/text_input_layout">
+
+        <android.support.design.widget.TextInputEditText
+            android:id="@+id/input_filesystem_username"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:inputType="text"
+            android:text="@string/default_username"
```

Comment:
Let's set these dynamically based on the filesystem, rather than a string resource. That way we only have to change it in one place if we decide to.

Reply:
Could you go into a bit more detail about this?

Reply:
It looks like you're actually already doing that in `setupTextInputs()`, sorry. But that means you should be able to remove the static xml attributes and therefore the string resources as well I believe.